### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by eranrund**

### CODE REVIEW

File: validator/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        --validator insecure-validator://127.0.0.1:5554/
+        --fog-ingest-enclave-css $(pwd)/ingest-enclave.css
+```
+
+Notice how `--validator` replaced `--peer` and `--tx-source-url`.
+
+
+## TLS between full-service and LVN
+
+The GRPC connection between `full-service` and `mc-ledger-validator` can optionally be TLS-encrypted. If you wish to use TLS for that, you'll a certificate file and the matching private key for it. For testing purposes you can generate your own self-signed certificate:
```
Comment (garbageslam) [inline validator/README.md:57]:
```suggestion
The GRPC connection between `full-service` and `mc-ledger-validator` can optionally be TLS-encrypted. If you wish to use TLS for that, you'll need a certificate file and the matching private key for it. For testing purposes you can generate your own self-signed certificate:
```

File: validator/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        --ledger-db /tmp/ledger-db/ \
+        --peer mc://node1.test.mobilecoin.com/ \
+        --peer mc://node2.test.mobilecoin.com/ \
+        --tx-source-url https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node1.test.mobilecoin.com/ \
+        --tx-source-url https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node2.test.mobilecoin.com/ \
+        --listen-uri "validator://0.0.0.0:5554/?tls-chain=server.crt&tls-key=server.key"
+```
+Notice that the `--listen-uri` argument has changed and points to the key and certificate you generated.
+
+Once the LVNis running, you will need to run `full-service`:
```
Comment (garbageslam) [inline validator/README.md:97]:
```suggestion
Once the LVN is running, you will need to run `full-service`:
```

File: validator/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+```sh
+    mkdir -p /tmp/wallet-db/
+    ./target/release/full-service \
+        --wallet-db /tmp/wallet-db/wallet.db \
+        --ledger-db /tmp/ledger-db/ \
+        --validator "validator://localhost:5554/?ca-bundle=server.crt&tls-hostname=localhost"
```
Comment (joekottke) [inline validator/README.md:103]: You might want to be consistent in localhost vs 127.0.0.1.  Up to you.

File: validator/README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Now, you can run the LVN with TLS enabled:
+```sh
+./target/release/mc-validator-service \
+        --ledger-db /tmp/ledger-db/ \
+        --peer mc://node1.test.mobilecoin.com/ \
+        --peer mc://node2.test.mobilecoin.com/ \
+        --tx-source-url https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node1.test.mobilecoin.com/ \
+        --tx-source-url https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node2.test.mobilecoin.com/ \
+        --listen-uri "validator://0.0.0.0:5554/?tls-chain=server.crt&tls-key=server.key"
```
Comment (joekottke) [inline validator/README.md:93]: Same comment here about listening on 127.0.0.1

**Review by joekottke — CHANGES_REQUESTED** [review]
Initial pass with some eyeballing.  Might have more on a second-pass, but otherwise it looks very thorough.

**Review by joekottke — APPROVED** [review]
LGTM

**Pull request merged**